pages/index.js

```diff
@@ -36,21 +84,41 @@ export default function Home() {
         </div>
       </div>
       <div className="container flex flex-wrap">
-        <div className="mt-4 mb-4 grid lg:grid-cols-3 sm:grid-cols-2 grid-flow-row gap-3 md:gap-10">
-          {data.map(function (d, idx) {
-            return (
-              <TopicBox
-                key={idx}
-                title={d.title}
-                body={d.body}
-                image={d.image}
-                imgalt={d.imgalt}
-                url={d.url}
-                subtopics={d.subtopics}
-              />
-            );
-          })}
-        </div>
+        {ADOBE_ENABLED === true ? (
+          <div className="mt-4 mb-4 grid lg:grid-cols-3 sm:grid-cols-2 grid-flow-row gap-3 md:gap-10">
+            {topicsData.map(
+              ({ title, body, image, imgalt, url, subtopics }, idx) => {
+                return (
+                  <TopicBox
+                    key={idx}
+                    title={title}
+                    body={body}
+                    image={image}
+                    imgalt={imgalt}
+                    url={url}
+                    subtopics={subtopics}
+                  />
+                );
+              }
+            )}
+          </div>
+        ) : (
```

Comment:
We pretty much have the same data here, whether it comes from the API or not. The difference is just whether we use `topicsData.map` or `data.map`. I think instead we should populate the "data" variable, so the page doesn't know the difference, and then not have an "if" switch here at all.